config.js

```diff
@@ -9,4 +9,4 @@ const { DB_PASSWORD } = process.env;
 
 module.exports.PORT = process.env.PORT || 3000;
 
-module.exports.DATABASE_URL = `mongodb://DND:${DB_PASSWORD}@dnd-shard-00-00-yxink.azure.mongodb.net:27017,dnd-shard-00-01-yxink.azure.mongodb.net:27017,dnd-shard-00-02-yxink.azure.mongodb.net:27017/mesto4?ssl=true&replicaSet=DND-shard-0&authSource=admin&retryWrites=true&w=majority`;
+module.exports.DATABASE_URL = DB_PASSWORD ? `mongodb://DND:${DB_PASSWORD}@dnd-shard-00-00-yxink.azure.mongodb.net:27017,dnd-shard-00-01-yxink.azure.mongodb.net:27017,dnd-shard-00-02-yxink.azure.mongodb.net:27017/mesto4?ssl=true&replicaSet=DND-shard-0&authSource=admin&retryWrites=true&w=majority` : 'mongodb://localhost:27017/mydb';
```

Comment:
Можно лучше: указать адрес для подключения к базе данных полностью.
Так как пароль можно передавать вместе с адресом, тогда можно адрес для подключения к БД полность указать в `.env`. Тогда в кнфиге будет:
```suggestion
module.exports.DATABASE_URL = process.env.DATABASE_URL || 'mongodb://localhost:27017/mydb';
```

Либо есть для продакшена то:
```suggestion
const isProduction = process.env.NODE_ENV === 'production';

module.exports.DATABASE_URL = isProduction ? process.env.DATABASE_URL : 'mongodb://localhost:27017/mydb';
```

Reply:
Исправил